inst/rsn/RSN_02.Rmd

```diff
@@ -0,0 +1,390 @@
+V(ws)$chicken<-sample(lc_eth_spdf@data$chicken, nrow(lc_eth_spdf@coords))
+
+```
+
+### Compare clustering coefficients
+
+```{r rsn_clust}
+
+data.frame(Model=c("Real World", "Small World"), 
+           Value=round(c(transitivity(rws), transitivity(ws)), 3),
+           Meaning=c("Low clustering", "Higher clustering"))
+
+```
+
+### Compare path lengths between all pairs of vertices
+
+```{r rsn_dist}
+
+data.frame(Model=c("Real World", "Small World"), 
+           Value=round(c(mean_distance(rws), mean_distance(ws)), 3))
+
+```
+
+### Compare closeness centrality
+
+Closeness centrality is defined as how many steps are required to access every other vertex from a given vertex.
+
+```{r rsn_close}
+
+data.frame(Model=c("Real World", "Small World"), 
+           Value=round(c(mean(closeness(rws)), mean(closeness((ws)))), 6))
+
+```
+
+### Compare assortativity based on chicken variable
+
+```{r rsn_assort}
+
+data.frame(Model=c("Real World", "Small World"), 
+           Value=round(c(assortativity(rws, V(rws)$chicken, directed=F),
+                         assortativity(ws, V(ws)$chicken, directed=F)), 6),
+           Meaning=c("Negative tendency of node connections where chicken values are similar", "Negative tendency of node connections where chicken values are similar"))
+
+```
+
+### Communities: Small World Network
+
+```{r community_ws}
+
+cfg<-cluster_fast_greedy(ws)
+clp<-cluster_label_prop(ws)
+kc <- coreness(ws, mode="all")   
+
+## GENERATE PLOT OF COMMUNITIES PER ALGORITHM
+
+layout(matrix(c(1,2,3,4), 2, 2, byrow = TRUE))
+par(mar=c(0,0,2,0), oma=c(0,0,0,0))
+plot(cfg, ws, main="Greedy Optimization", edge.arrow.size=.2,
+     vertex.label.color="transparent", vertex.color="transparent",
+     vertex.frame.color="transparent")
+plot(clp, ws, main="Propagating Labels", edge.arrow.size=.2,
+     vertex.label.color="transparent", vertex.color="transparent",
+     vertex.frame.color="transparent")
+plot(ws, vertex.size=kc, main="K-Core Decomp", edge.arrow.size=.2,
+    vertex.label.color="transparent", vertex.color="transparent",
+     vertex.frame.color="transparent")
+
+```
+
+### Communities: Real World Network
+
+```{r community_rws}
+
+cfg<-cluster_fast_greedy(as.undirected(rws))
+clp<-cluster_label_prop(rws)
+kc <- coreness(rws, mode="all")   
+
+## GENERATE PLOT OF COMMUNITIES PER ALGORITHM
+
+layout(matrix(c(1,2,3,4), 2, 2, byrow = TRUE))
+par(mar=c(0,0,2,0), oma=c(0,0,0,0))
+plot(cfg, rws, main="Greedy Optimization", edge.arrow.size=.2,
+     vertex.label.color="transparent", vertex.color="transparent",
+     vertex.frame.color="transparent")
+plot(clp, rws, main="Propagating Labels", edge.arrow.size=.2,
+     vertex.label.color="transparent", vertex.color="transparent",
+     vertex.frame.color="transparent")
+plot(rws, vertex.size=kc, main="K-Core Decomp", edge.arrow.size=.2,
+    vertex.label.color="transparent", vertex.color="transparent",
+     vertex.frame.color="transparent")
+
+```
```

Comment:
I think these plots will be much more intuitive if we project them onto space, as you did in the first Rmd.  It might be necessary to make them in large format rather than the small size that come out with Rmd plot defaults.